### ISSUE TITLE
[BLOCKED] sunset preservation usage of the NetApp 8040

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/preservation_catalog#1464]
see https://github.com/sul-dlss/preservation_catalog/issues/1327#issuecomment-610702636

- [ ] 1.  confirm that we have copied all of the preservation storage root content from the 8040 netapp filer to the new storage brick.
- [ ] 2. for all of the storage roots on the hardware to be retired:
  - [ ] all of the production preservation_catalog DB records for moabs that have been copied to the new storage hardware have been migrated to point to the moabs' new home on the storage brick.  this migration is done in bulk for each migrated storage root using a `rake` task (see https://github.com/sul-dlss/preservation_catalog/blob/master/.github/ISSUE_TEMPLATE/storage-migration-checklist.md).
  - [ ] infra dev and ops have confirmed, via e.g. checksum validation of all moved moabs (as outlined in the aforementioned checklist) that the copied content landed safely on the new storage, with no degradation compared to the original moabs.  the preservation catalog checksum validation code will attempt to confirm that the catalog accurately reflects the state of things on disk (and will warn us if that's not the case)
- [ ] 3. our repository manager (@andrewjbtw) is satisfied that the migrated preservation content has landed safely in its new home and that the preservation_catalog records for the migrated moabs have also been properly updated to point to the new storage location.  i assume this confirmation will happen via a combination of discussing the migration attempts with devs/ops, and spot checking results himself, but he can correct or flesh out this part as needed.

**Comment by jmartin-sul** [comment]
blocked by need to actually do the prod migration for this NetApp filer

**Comment by jmartin-sul** [comment]
turns out that the migrations aren't divided by underlying storage hardware, and the first migration contained storage roots from both the 6220 and the 8040.

closing in favor of #1555, #1556, and #1557
